### ISSUE TITLE
FullPageScreenshot with Highlighted Headers

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#139]
Screengrab/videos (insert file names here)
2020 08 18, FullPageScreenshot with Highlighted Headers.png

Link
https://d8-stg.boston.gov

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > macOS > Chrome 84

Trusted Tester Test
Headings

Brief Description
The section with today’s date is not marked as a heading

Steps to reproduce
1)	Load the page
2)	Note that the section with today’s date looks like a heading (“Wednesday, August 19” on this image), but it is not marked as a heading.
a)	Bug:  This makes it more difficult for a screen reader to identify this section
3)	Note:  other sections are marked as headings, such as:
a)	Common Resources
b)	Latest News
c)	Featured Videos
d)	BOS:311 Service Requests
e)	Stay Connected
f)	NOTE >>> These are highlighted in Greed in the referenced image
![2020 08 18, FullPageScreenshot with Highlighted Headers](https://user-images.githubusercontent.com/33845414/91489507-ee508880-e87e-11ea-81cd-d6e386e2bac4.png)

Desired Result
All sections that visually look like headings should be marked as a heading in html

**Comment by subaha-cob** [comment]
Test at: https://d8-uat.boston.gov/

**Comment by david-iterators** [comment]
Verified on UAT.
